src/constants.js

```diff
@@ -24,10 +24,34 @@ constants.minBallotDurationInDays = 2
 constants.startTimeOffsetInMinutes = 5
 constants.endTimeDefaultInMinutes = 2890
 constants.getTransactionReceiptInterval = 5000
-constants.NETID_SOKOL = '77'
-constants.NETID_CORE = '99'
-constants.NETID_DAI_TEST = '79'
-constants.NETID_DAI = '100'
+
+constants.NETWORKS = {
+  '77': {
+    NAME: 'Sokol',
+    RPC: 'https://sokol.poa.network',
```

Comment:
@varasev where do we use RPC endpoints of chains?

Reply:
@vbaranov It's just a copy-paste from Validators DApp where they are used. Voting DApp doesn't use them. Forgot to remove. I'll fix this.